raiden/transfer/unlock.py

```diff
@@ -0,0 +1,141 @@
+from typing import Optional
+
+from eth_utils import to_checksum_address, to_hex
+
+from raiden.constants import EMPTY_HASH
+from raiden.exceptions import RaidenUnrecoverableError
+from raiden.storage.restore import channel_state_until_state_change
+from raiden.transfer.channel import get_batch_unlock_gain
+from raiden.transfer.identifiers import CanonicalIdentifier
+from raiden.transfer.state import ChainState, NettingChannelState, NettingChannelEndState
+from raiden.transfer.utils import get_state_change_with_balance_proof_by_locksroot, \
+    get_event_with_balance_proof_by_locksroot
+from raiden.transfer.views import get_channelstate_by_token_network_and_partner
+from raiden.utils import Address
+from raiden.utils.typing import TokenNetworkID
+
+def get_channel_state(
```

Comment:
Sounds that we already have this in another place, maybe views.py or channel.py but i think i saw something similar somewhere else

Reply:
@marcosmartinez7 told me the same. I'm sure you are referreing to `views.get_channelstate_by_token_network_and_partner` or some similar function, but they just return None.

This code is just an encapsulation of what already existed  in raiden in unlock handler, which I found a bit confusing as whole, so preferred not to play much with it for now.